### ISSUE TITLE
Avoid sys.exit - raise exceptions

### ISSUE THREAD

**Issue by yarikoptic** [vsoch/qme#34]
BTW if there is any intent for this functionality to be used as a library by other projects, `sys.exit` should not be used, and rather regular (ValueError, RuntimeError, IOError) and/or custom (e.g. Job status etc oriented) exceptions should be raised  .  Then somewhere on the top in cmdline interface there should be a handler which would redirect them into `sys.exit(str(exc))` or alike.

**Comment by vsoch** [comment]
This is a good idea - it would be nice to have named exceptions like I've seen in other libraries that also giving meaningful feedback about the errors.

**Comment by vsoch** [comment]
This will be addressed after #31.

**Comment by vsoch** [comment]
I'm having too much fun doing this. :P 
![Screenshot from 2020-05-31 15-41-10](https://user-images.githubusercontent.com/814322/83363275-6decbb00-a355-11ea-9783-47ee9e305703.png)

**Comment by yarikoptic** [comment]
Indeed too much ;-) note that exceptions become part of the API. See recent or/discussion with good pointers in pybids https://github.com/bids-standard/pybids/pull/615 . 
Exceptions which come about from incorrect user specification of the value are typically a subclass of ValueError (said Yarik, but he forgot if KeyError is such or just an Exception)

**Comment by vsoch** [comment]
@yarikoptic see https://github.com/vsoch/qme/pull/36.
